homeassistant/components/bang_olufsen/media_player.py

```diff
@@ -309,18 +309,18 @@ async def _update_playback_metadata(self, data: PlaybackContentMetadata) -> None
 
         self.async_write_ha_state()
 
-    async def _update_playback_error(self, data: PlaybackError) -> None:
+    def _update_playback_error(self, data: PlaybackError) -> None:
```

Comment:
https://developers.home-assistant.io/docs/asyncio_thread_safety

```suggestion
    def _async_update_playback_error(self, data: PlaybackError) -> None:
```

We usually prefix functions decorated with `@callback` with `async_` since we have both sync and async code in this codebase and it helps lead to less mistakes when we have naming conventions that provide clues

Reply:
I've added the _async prefix where applicable.